core/vulkan/perfetto_producer/perfetto_threadlocal_emitter.inc

```diff
@@ -18,180 +18,226 @@
+      interned_categories_(&arena_), interned_function_names_(&arena_),
+      interned_vulkan_annotation_keys_(&arena_), last_reset_timestamp_(0),
+      reset_period_ms_(SEQUENCE_INCREMENTAL_DATA_RESET_DURATION_MS),
+      reset_(false) {
+  thread_name_ = Thread::current().get_name();
+  process_name_ = core::get_process_name();
+  thread_id_ = Thread::current().id();
+  process_id_ = core::get_process_id();
+  PerfettoProducer<T>::Get().RegisterEmitter(this);
 }
 
-template<typename T>
+template <typename T>
 ThreadlocalEmitter<T>::~ThreadlocalEmitter() {
-    PerfettoProducer<T>::Get().UnregisterEmitter(this);
+  PerfettoProducer<T>::Get().UnregisterEmitter(this);
 }
 
-template<typename T>
+template <typename T>
 uint64_t ThreadlocalEmitter<T>::InternName(
-        const char* name,
-        typename PerfettoProducer<T>::TraceContext::TracePacketHandle& packet,
-        perfetto::protos::pbzero::InternedData** interned_data) {
-    uint64_t val = interned_names_.findOrZero(name);
-    if (!val) {
-        val =  interned_names_.count() + 1;
-        interned_names_[name] = val;
-        if (!*interned_data) {
-            *interned_data = packet->set_interned_data();
-        }
-        perfetto::protos::pbzero::EventName* event_name =
-        (*interned_data)->add_event_names();
-        event_name->set_iid(val);
-        event_name->set_name(name);
-        return interned_names_.count();
+    const char *name,
+    typename PerfettoProducer<T>::TraceContext::TracePacketHandle &packet,
+    perfetto::protos::pbzero::InternedData **interned_data) {
+  uint64_t val = interned_names_.findOrZero(name);
+  if (!val) {
+    val = interned_names_.count() + 1;
+    interned_names_[name] = val;
+    if (!*interned_data) {
+      *interned_data = packet->set_interned_data();
     }
-    return val;
+    perfetto::protos::pbzero::EventName *event_name =
+        (*interned_data)->add_event_names();
+    event_name->set_iid(val);
+    event_name->set_name(name);
+    return interned_names_.count();
+  }
+  return val;
 }
 
-template<typename T>
+template <typename T>
 uint64_t ThreadlocalEmitter<T>::InternAnnotationName(
-        const char* name,
-        typename PerfettoProducer<T>::TraceContext::TracePacketHandle& packet,
-        perfetto::protos::pbzero::InternedData** interned_data) {
-    uint64_t val = interned_annotation_names_.findOrZero(name);
-    if (!val) {
-        val =  interned_annotation_names_.count() + 1;
-        interned_annotation_names_[name] = val;
-        if (!*interned_data) {
-            *interned_data = packet->set_interned_data();
-        }
-        auto* event_name = (*interned_data)->add_debug_annotation_names();
-        event_name->set_iid(val);
-        event_name->set_name(name);
-        return interned_annotation_names_.count();
+    const char *name,
+    typename PerfettoProducer<T>::TraceContext::TracePacketHandle &packet,
+    perfetto::protos::pbzero::InternedData **interned_data) {
```

Comment:
Something bad happened to formatting, specifically *

Reply:
This file is clang-formatted now, may be that's why.

Reply:
It looks to be the wrong clang-formatting rules however?

Reply:
Strange. Not sure how this happened. Fixed it.